### PR TITLE
Add `lineax`

### PR DESCRIPTION
**Pull request by dhirschfeld**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by dhirschfeld** [comment]
Windows is skipped upstream:
https://github.com/conda-forge/jaxlib-feedstock/blob/main/recipe/meta.yaml#L23

**Comment by dhirschfeld** [comment]
@conda-forge/core I think this is good to merge. Windows fails as it can't be installed as the upstream dep isn't being built for the Windows platform:
```
nothing provides jaxlib >=0.4.11 needed by jax-0.4.13-pyhd8ed1ab_0
```

I think we still want to build a `noarch` recipe here as it will Just Work as soon as upstream publishes a build of `jaxlib` for Windows.

Until that time the `noarch` package wouldn't be able to be installed on Window (due to the missing dep) but would work fine on all other platforms where `jaxlib >=0.4.11` is available.

File: recipes/lineax/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.9,<4.dev0
+    - hatchling
+    - pip
+  run:
+    - python >=3.9
```
Comment (ocefpaf) [inline recipes/lineax/meta.yaml:23]: While I don't think that is needed, it should be the same as above.

```suggestion
    - python >=3.9,<4.dev0
```

Comment (ocefpaf) [reply recipes/lineax/meta.yaml]: It doesn't hurt b/c I believe the upper pin is unnecessary. I've seen this bug in grayskull before BTW, but I was too lazy to report ;-p

**Pull request merged**